os-hardening/templates/limits.conf.tmpl

```diff
@@ -1,4 +1,4 @@
-# {{ pillar['message_do_not_modify'] }}
+# {{ salt['pillar.get']('message_do_not_modify','MANAGED BY SALT - DO NOT MODIFY') }}
```

Comment:
Sure - or we can remove it i just kept it because i didn't want to change the current functionality.
My change was mainly made to ensure that the state works without people setting something from the pillar